roles/openshift_logging_eventrouter/templates/eventrouter-template.j2

```diff
@@ -65,7 +65,7 @@ objects:
               imagePullPolicy: IfNotPresent
               resources:
                 limits:
-{% if cpu_limit is defined %}
+{% if cpu_limit is not none %}
```

Comment:
can you make this `if cpu_limit is defined and cpu_limit is not none and cpu_limit != ''` ?

Reply:
`cpu_limit` would always be defined - its being passed to the task

I could set `cpu_limit: {{ openshift_logging_eventrouter_cpu_limit | default('') }}` and leave just the `cpu_limit != ''` instead, does that sound good?

Reply:
if its always defined, then that `| default('')` wouldn't trigger. what about just checking for `not none and cpu_limit != ''` ?

Reply:
Updated to include `cpu_limit != ''`